change/@fluentui-react-carousel-a4f2fe14-2f61-4182-81d9-df0672b90ed5.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33015&commitId=f12ff3634695ca37061677c8d76127edea8c1dbc&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|4| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay RTL.chromium.png|1166| Changed|
|Drawer.overlay drawer full.chromium.png|2859| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>